samples/server.c

```diff
@@ -217,26 +215,28 @@ static void do_dma_io(vfu_ctx_t *vfu_ctx, struct server_data *server_data)
 
     assert(vfu_ctx != NULL);
 
-    ret = vfu_addr_to_sg(vfu_ctx, server_data->regions[0].addr, count, &sg,
-                         1, PROT_WRITE);
+    ret = vfu_addr_to_sg(vfu_ctx,
+                         (vfu_dma_addr_t)server_data->regions[0].iova.iov_base,
+                         count, &sg, 1, PROT_WRITE);
     if (ret < 0) {
-        errx(EXIT_FAILURE, "failed to map %#lx-%#lx: %s\n",
-             server_data->regions[0].addr,
-             server_data->regions[0].addr + count -1, strerror(-ret));
+        errx(EXIT_FAILURE, "failed to map 0x%p-0x%p: %s\n",
```

Comment:
I check with gcc 8.3.0 and `%p` add the leading `0x`.

Reply:
oops, fixed the other spot too